### ISSUE TITLE
Feature Request: Find Recently Modified

### ISSUE THREAD

**Issue by metal450** [giuspen/cherrytree#950]
Apologies if this already exists somewhere, but if so I wasn't able to find it:

Is there some way to find "recently modified nodes?"  I use this often in OneNote to i.e. see a "log" of what I've been working on, when using it as an issue tracker for personal software projects, etc.

Here's a screenshot of how it looks in OneNote: https://lnk2.tk/snaps/2020-06-21_12.00.08.png

If not, then please consider this a feature request :)

**Comment by giuspen** [comment]
No, you can currently only do a research filtering for modification time

**Comment by metal450** [comment]
Feature request, then :)
